packages/odyssey/src/scss/vendors-ext/_choices-ext.scss

```diff
@@ -255,35 +262,35 @@ $choices-icon-cross-inverse: url('data:image/svg+xml;base64,PHN2ZyB3aWR0aD0iMjEi
 
   .ods-select--item {
     display: inline-block;
-    margin-right: 3.75px;
-    margin-bottom: 3.75px;
-    padding: 4px $spacing-s;
-    border: 1px solid darken($color-primary-base, 5%);
-    border-radius: ($base-border-radius * 2);
-    background-color: $color-primary-base;
-    color: $white;
+    margin-right: $spacing-xs;
+    margin-bottom: $spacing-xs;
+    padding: 0 $spacing-s;
+    border: 1px solid $color-primary-light;
+    border-radius: 1em;
+    background-color: cv('blue', '000');
+    color: $text-body;
     font-size: $size-body-base;
-    font-weight: 500;
+    font-weight: 400;
+    line-height: calc((#{$base-line-height} * 1em) - 2px); // Adjust for pill border
     word-break: break-all;
     vertical-align: middle;
 
     &[data-deletable] {
-      padding-right: 5px;
+      padding-right: $spacing-xs;
     }
 
     [dir='rtl'] & {
       margin-right: 0;
-      margin-left: 3.75px;
+      margin-left: $spacing-xs;
     }
 
     &.is-ods-select-highlighted {
-      border: 1px solid darken($color-primary-base, 10%);
-      background-color: darken($color-primary-base, 5%);
+      background-color: $color-primary-light;
     }
 
     .is-ods-select-disabled & {
-      border: 1px solid darken(cv('gray', '300'), 35%);
-      background-color: darken(cv('gray', '300'), 25%);
+      border: 1px solid cv('gray', '500');
```

Comment:
Should you use `$border-color-ui` here?

Reply:
Yes! Excellent catch! I forgot to go back and refactor the Multi-Select coloration because they were parallel PRs. Fixing now, so `develop` can inherit when we merge this there as well.

Reply:
Fixed and pushed to preview!